build.gradle

```diff
@@ -2,8 +2,8 @@
 buildscript {
     repositories {
         google()
-        jcenter()
         mavenCentral()
+        jcenter()
```

Comment:
Just curious, does the order here matter?

Reply:
It will pull libraries from those repositories in that order. jcenter is deprecated to read only so we should eventually move away from pulling react-native from jcenter. Can't do without it just yet since though.